src/__tests__/load-modules.test.ts

```diff
@@ -55,7 +55,7 @@ describe('loadModules', function () {
     const deps = {
       container,
       listModules: jest.fn(() => moduleLookupResult),
-      require: jest.fn(async (p) => modules[path.parse(p).base]),
```

Comment:
This needs to stay async to be as close to the real behavior as possible. If we were to remove `await` at the call-site, this would give the wrong result.

Reply:
Got it, not removing it then.
Only updating the function declarations to arrow functions.